src/olympia/discovery/api_urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import include, url
+
+from rest_framework.routers import SimpleRouter
+
+from . import views
+
+# Because we're registering things at "/discovery/" we don't want the trailing
+# slash that comes with the Router by default.
```

Comment:
Ah, because it would result in `/v1/discovery//`, gotcha.

But we really miss the trailing slashes in case you do a sub-route, e.g

```python
discovery.register(r'thunderbird', views.ThunderbirdDiscoveryViewSet, base_name='discovery-thunderbird')
```

'cause it's a `prefix` `r'thunderbird/` should not work imho.

Imho it's best to include the urlpatterns here in the root like this:

```python
url(r'^v3/', include('olympia.discovery.api_urls')),
```

and then go ahead and define sub-routes in the discovery specific urls.py

```python
router = DefaultRouter()
router.register('discovery', views.DiscoveryViewSet, 'discovery')
```

And, if we need that, let's use drf-extensions: https://chibisov.github.io/drf-extensions/docs/#nested-routes